.eslintignore

```diff
@@ -0,0 +1,3 @@
+jspm.browser.js
```

Comment:
Mostly because they are generated/modified when using jspm. They are not really human-controlled.